src/wp-includes/class-wp-scripts.php

```diff
@@ -771,6 +788,23 @@ public function add_data( $handle, $key, $value ) {
 		return parent::add_data( $handle, $key, $value );
 	}
 
+	/**
+	 * Check all handles for any delayed inline scripts.
+	 *
+	 * @return bool True if script present. False if empty.
+	 */
+	public function maybe_has_delayed_inline_script() {
```

Comment:
@kt-12 this is a nitpick comment I guess, but I am wondering if `maybe` is the correct wording here, as it seems to be like you are _indefinitely_ checking of a deferred or async script has inline scripts? This is not a case of `maybe` and seems like it's more a definite outcome? 

If you agree, let's rename this to `has_delayed_inline_script()` as that is more true to the logic here.

If I've misunderstood, please comment as such thank you :)